tests/test_api.py

```diff
@@ -64,5 +63,4 @@ async def test_predict_endpoint(monkeypatch):
     monkeypatch.setenv("X_API_KEY", "test")
 
     output = await predict(appointments_pydantic, "2024-07-16", FakeDB(), "test")
-    output_df = pd.DataFrame(output)
-    assert output_df.shape == (5, 17)
+    assert "message" in output
```

Comment:
Do we test the output format somewhere else? Might nice to keep it in

Reply:
Yeah, true. I reverted this, so predict now behaves as it did previously